nUpdate.WPFUserInterface/ViewModel/ChangelogVm.cs

```diff
@@ -0,0 +1,160 @@
+                    : updateConfiguration.Changelog.First(item => item.Key.Name == "en").Value;
+
+                ChangelogText +=
+                    string.Format(string.IsNullOrEmpty(ChangelogText) ? "{0}:\n{1}" : "\n\n{0}:\n{1}",
+                        versionText, changelogText);
+            }
+            if (OperationAreas == null || OperationAreas.Count == 0)
+            {
+                AccessesText = $"{LocProperties.NewUpdateDialogAccessText} -";
+                return;
+            }
+
+            AccessesText =
+                $"{LocProperties.NewUpdateDialogAccessText} {string.Join(", ", LocalizationHelper.GetLocalizedEnumerationValues(LocProperties, OperationAreas.Cast<object>().GroupBy(item => item).Select(item => item.First()).ToArray()))}";
+            }
+
+        #endregion
+
+
+        #region "Properties"
+        public bool DialogResult { get; set; }
+        public string WindowTitle { get; set; }
+        public Dispatcher CurrentDispatcher { get; set; }
+        public void DialogLoaded(){}
+        public void DialogClosing(){}
+
+        public LocalizationProperties LocProperties { get; }
+        public List<OperationArea> OperationAreas { get; set; }
+        public string Header { get; }
+        public string InfoText { get; }
+        public string AviableVersionText { get; }
+        public string CurrentVersionText { get; }
+        public string AccessesText { get; }
+        public string ChangelogText { get; }
+        public string UpdateSizeText { get; }
+
+        #endregion
+
+
+
+        #region "Commands"
+
+        public ICommand DoInstallCommand { get; }
+    
+        private void DoInstallCommand_Execute()
+        {
+            if (!SizeHelper.HasEnoughSpace(UpdateManager.TotalSize, out var necessarySpaceToFree))
+            {
+                var packageSizeString = SizeHelper.ConvertSize((long) UpdateManager.TotalSize);
+                var spaceToFreeString = SizeHelper.ConvertSize((long) necessarySpaceToFree);
+                
+                var msgService = ServiceContainer.Instance.GetService<IMessageboxService>();
+                msgService.Show($"You don't have enough disk space left on your drive and nUpdate is not able to download and install the available updates ({packageSizeString}). Please free a minimum of {spaceToFreeString} to make sure the updates can be downloaded and installed without any problems."
+                    , "Not enough disk space.", EnuMessageBoxButton.Ok,
+                    EnuMessageBoxImage.Warning);
+                return;
+            }
+            DialogResult = true;
+            var dialogService = ServiceContainer.Instance
+                .GetService<IDialogWindowService>();
+            dialogService.CloseDialog();
+        }
+
+
+    
+
+        public ICommand AbortCommand { get; }
+
+        private void AbortCommand_Execute()
```

Comment:
Rename the command execution methods simply to their action: "private void Abort()" e.g.